js/assessments/wordComplexityAssessment.js

```diff
@@ -0,0 +1,66 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var filter = require( "lodash/filter" );
+
+/**
+ * Calculates the complexity of the text based on the syllables in words.
+ * @param {number} wordCount The number of words used.
+ * @param {number} tooComplexWords The number of words with 4 or more syllables.
+ * @param {number} recommendedValue The recommendedValue for amount of syllables.
+ * @param {object} i18n The object used for translations.
+ * @returns {{score: number, text}} resultobject with score and text.
+ */
+var calculateComplexity = function( wordCount, tooComplexWords, recommendedValue, i18n ) {
+	var percentage = ( tooComplexWords / wordCount ) * 100;
+	var recommendedMaximum = 10;
+	// 6 is the number of scorepoints between 3, minscore and 9, maxscore. For scoring we use 10 steps.
+	// Up to 6.7 percent is for scoring a 9, higher percentages give lower scores.
+	var score = 9 - Math.max( Math.min( ( 6 / 10 ) * ( percentage - 6.7 ), 6 ), 0 );
+	if ( score >= 7 ) {
+		return {
+			score: score,
+			// translators: %1$d expands to the number of too complex words, %2$d expands to the recommended number of syllables
```

Comment:
I believe these translator strings need to be moved to be directly above the actual translate.

This also applies for the translator string on line 31.